paddlecloud/paddlecloud/urls.py

```diff
@@ -36,6 +36,8 @@
     url(r"^api/v1/token2user/", paddlejob.views.GetUserView.as_view()),
     url(r"^api/v1/filelist/", paddlejob.views.SimpleFileList.as_view()),
     url(r"^api/v1/registry/", paddlejob.registry.RegistryView.as_view()),
+    url(r"^api/v1/publish/", paddlejob.views.FilePublishAPIView.as_view()),
+    url(r"^filepub/", paddlejob.views.file_publish_view),
```

Comment:
Why does not use the same URL prefix?

Reply:
One for `wget` to download and one for paddle cloud client API calls.